packages/battery_plus/battery_plus/example/integration_test/battery_plus_test.dart

```diff
@@ -7,7 +7,6 @@
 import 'dart:io';
 
 import 'package:battery_plus/battery_plus.dart';
-import 'package:flutter/foundation.dart';
```

Comment:
This one can't be dropped even though the action reports so. If you check the code in your IDE you would see that there is usage of `debugPrint()` and this is why `foundation` is imported.

Reply:
Based on this I expect that some other reported issues might also be false positive.

Reply:
I don't find any `debugPrint` in whole `plus_plugins`?

<img width="831" alt="Screen Shot 2022-09-28 at 10 30 15" src="https://user-images.githubusercontent.com/43643339/192729885-d70de255-de9f-417b-95db-8c8d42777594.png">

Reply:
Looks like you are searching it wrong. In the same file the report is.
<img width="576" alt="Screenshot 2022-09-28 at 12 33 46" src="https://user-images.githubusercontent.com/13467769/192744709-08d502df-1561-4950-b690-1c7491e2ccfb.png">

Reply:
This was recently removed (it was me), probably you have an outdated main branch

Reply:
It is [not](https://github.com/fluttercommunity/plus_plugins/search?q=debugPrint) in this file or repo.